GOVERNANCE.md

```diff
@@ -0,0 +1,104 @@
+  maintainer. Becoming a maintainer generally means that you are going to be spending substantial
+  time (>25%) on Envoy for the foreseeable future. You should have domain expertise and be extremely
+  proficient in C++. Ultimately your goal is to become a senior maintainer that will represent your
+  organization.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance
+  of the existing senior maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews
+  for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of
+  community reviews).
+* After a period of approximately 2-3 months of working together and making sure we see eye to eye,
+  the existing senior maintainers will confer and decide whether to grant maintainer status or not.
+  We make no guarantees on the length of time this will take, but 2-3 months is the approximate
+  goal.
+
+## Your organization is currently a maintainer
+
+* First decide whether your organization really needs more people with maintainer access. Valid
+  reasons are "blast radius", a large organization that is working on multiple unrelated projects,
+  etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard"
+  maintainer access.
+* "Standard" maintainer access can be upgraded to "senior" maintainer access after another 1-2
+  months of work and another conference of the existing senior committers.
+
+## Maintainer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+* During GitHub issue triage, apply all applicable [labels](https://github.com/tikv/tikv/labels)
+  to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply
+  is somewhat subjective so just use your best judgment. A few of the most important labels that are
+  not self explanatory are:
+  * **beginner**: Mark any issue that can reasonably be accomplished by a new contributor with
+    this label.
+  * **help wanted**: Unless it is immediately obvious that someone is going to work on an issue (and
+    if so assign it), mark it help wanted.
+  * **question**: If it's unclear if an issue is immediately actionable, mark it with the
+    question label. Questions are easy to search for and close out at a later time. Questions
+    can be promoted to other issue types once it's clear they are actionable (at which point the
+    question label should be removed).
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.

```

Comment:
"or closing them" is not clear to me.

Reply:
Closing (the PR)

Reply:
As in "closing the PR if they are not moving at the right pace?"

Reply:
👍

Reply:
Then the sentence should be “Make sure that ongoing PRs are moving forward at the right pace, or close them if they are not."